### ISSUE TITLE
{Linked:Bug2920242;Linked:Bug2920139} [Adaptive Cards>Designer]: 'You are invited to a swarm session' is not defined as a heading programmatically.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8572]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/designer/
OS Version: 23H2{OS Build 25387.1200}
Browser Version: Version 116.0.1908.0 (Official build) dev (64-bit)
Screen Reader: Narrator

**Repro Steps:**
1. Open the above URL in the edge dev browser.
2. Navigate to the 'Card Payload Editor' expand button and press enter key to activate it.
3. Paste the below attached JSON.
4. Navigate to Preview button and press enter key to activate it.
5. Navigate the opened dialog and observe the issue.

**Actual Result:**
You are invited to a swarm session' is not defined as a heading programmatically. It is defined under paragraph.

Expected Result:
**You are invited to a swarm session'** should be as a heading programmatically and when focus lands on it screen reader should convey the heading level as 1.

**User Impact:**
It will impact the screen reader user as they will not able to understand whether there is a heading or not?

**WCAG Reference:**
[Understanding Success Criterion 1.3.1: Info and Relationships | WAI | W3C](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships)

Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0)

### Screenshots

https://github.com/microsoft/AdaptiveCards/assets/95913748/cb5e4e0f-2728-4706-86f0-152d64ac00c0

!['You are invited to a swarm session is not defined as heading](https://github.com/microsoft/AdaptiveCards/assets/95913748/6576d870-247f-4162-8860-182f8c5bd80a)

### Card JSON

```JSON
{
    "type": "AdaptiveCard",
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
    "version": "1.5",
    "body": [
        {
            "type": "TextBlock",
            "text": "You are invited to a swarm session",
            "wrap": true,
            "weight": "Bolder",
            "size": "Large"
        },
        {
            "type": "RichTextBlock",
            "inlines": [
                {
                    "type": "TextRun",
                    "text": "Mane Koa ",
                    "weight": "Bolder"
                }
            ]
        },
        {
            "type": "TextBlock",
            "text": "Title:",
            "wrap": true,
            "weight": "Bolder"
        },
        {
            "type": "TextBlock",
            "text": "The customer is having trouble installing their air conditioning unit to their power supply",
            "wrap": true
        },
        {
            "type": "TextBlock",
            "text": "Description:",
            "wrap": true,
            "weight": "Bolder"
        },
        {
            "type": "TextBlock",
            "text": "I am having a hard time figuring out what the problem is with this install on the customer air conditioning unit. Here are some of the things we have tried: \n Include details about your case, describe expected and results, include things you've tried",
            "wrap": true
        },
        {
            "type": "TextBlock",
            "text": "You have been asked to join because of these skills: ",
            "wrap": true,
            "weight": "Bolder"
        },
        {
            "type": "TextBlock",
            "text": "Litware, Cafe100",
            "wrap": true
        },
        {
            "type": "ActionSet",
            "actions": [
                {
                    "type": "Action.Submit",
                    "title": "Accept",
                    "data": {
                        "button": "accept"
                    }
                },
                {
                    "type": "Action.Submit",
                    "title": "Reject",
                    "data": {
                        "button": "reject"
                    }
                }
            ]
        }
    ]
}
```

**Comment by vagpt** [comment]
This bug is originally tracking with using URL below.

https://dev.azure.com/dynamicscrm/OneCRM/_workitems/edit/2920139
https://dev.azure.com/dynamicscrm/OneCRM/_workitems/edit/2920242

Also, changing it from Usable to MAS as per the both linked bugs.

**Comment by vagpt** [comment]
Since the both linked bugs are marked as closed and having A11yTTValidated tag, hence closing this issue.
